### ISSUE TITLE
Accordions, keyboard-accessibility

### ISSUE THREAD

**Issue by aceyoung9** [HackBeanpot/build4blm#71]
Hi friends,

I'm a HackBeanpot Core team alum and a front-end dev who prefers to navigate with a keyboard only. (for accessibility reasons; my wrist tendons do not get along with mice)

### Issues observed:

* The accordions aren't reachable using the Tab key
  * this can likely be fixed with using a `<button>` element instead of a `<div class="accordion-header">`
* they are also missing some aria roles to describe whether a section is expanded/collapsed. This is likely leading to an inadequate experience for screen readers.

### Resources:

* https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction
* https://inclusive-components.design/collapsible-sections/

I'm happy to send in a PR addressing this, or help test a branch if someone on the team would like to use this issue as a front-end learning opportunity.

Thanks! 😸

**Comment by feliciazhang** [comment]
Thanks so much for pointing this out! @daniaalnahdi will take this on :)

**Comment by feliciazhang** [comment]
@aceyoung9 Would you mind putting up a PR for this actually if you have the time? We want to test run having external contributions (you'll be the first!) Thanks!

**Comment by aceyoung9** [comment]
@feliciazhang I'll give it a shot after work today. Thanks!
